core/server/data/migrations/versions/5.0/2022-04-25-10-32-backfill-mrr-for-discounted-subscriptions.js

```diff
@@ -0,0 +1,44 @@
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const subscriptionsToUpdate = await knex('members_stripe_customers_subscriptions AS s')
+            .join('offers AS o', 's.offer_id', '=', 'o.id')
+            .where('o.duration', '=', 'forever')
+            .andWhere('s.mrr', '!=', '0')
```

Comment:
Should `'0'` be a string? 🤔 Knex pumps out the following:

```
  knex:query select `s`.*, `o`.`discount_type` as `offer_type`, `o`.`discount_amount` as `offer_amount` from `members_stripe_customers_subscriptions` as `s` inner join `offers` as `o` on `s`.`offer_id` = `o`.`id` where `o`.`duration` = ? and `s`.`mrr` != ? trx5 +1ms
  knex:bindings [ 'forever', '0' ] trx5 +1ms
```

I'm going to guess MySQL/SQLite will convert it but I'd prefer not to leave it to the engine

Reply:
Yeah fair, I'll update that!